src/views/LoginPage.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { Alert, AlertTitle, Box, Container } from '@mui/material';
+import Button, { ButtonProps } from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import TextField from '@mui/material/TextField';
+import { Column } from '../components/Column';
+import theme from '../themes/theme';
+import { authenticateUser } from '../utils/authenticateUser';
+
+const ColorButton = styled(Button)<ButtonProps>(() => ({
+  backgroundColor: theme.palette.primary.main,
+  '&:hover': {
+    backgroundColor: theme.palette.primary.dark
+  }
+}));
+
+const StyledPaper = styled(Paper)(() => ({
+  width: '359px',
+  height: '261px',
+  display: 'flex',
+  padding: '40px'
+}));
+
+const StyledContainer = styled(Container)(() => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  height: '100vh',
+  width: '100vw',
+  flexGrow: 1,
+  backgroundColor: theme.palette.info.main
+}));
+
+export const LoginPage = () => {
+  const [username, setUsername] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [error, setError] = useState<boolean>(false);
+
+  const handleSubmit = async () => {
+    try {
+      const token = await authenticateUser(username, password);
+      sessionStorage.setItem('token', token);
+      setUsername('');
```

Comment:
```
setUsername('');
setPassword('');
```
is the same for the happy and unhappy path.
You can create a `resetLoginForm` function and call it in the `finally` block.